api/qanpb/collector.proto

```diff
@@ -295,6 +295,34 @@ message MetricsBucket {
   float m_docs_scanned_min = 155;
   float m_docs_scanned_max = 156;
   float m_docs_scanned_p99 = 157;
+  //
+  // PostgreSQL metrics.
+  //
+
+  // Total number of shared block cache hits by the statement.
+  float m_shared_blks_hit = 200;
+  // Total number of shared blocks read by the statement.
+  float m_shared_blks_read = 201;
+  // Total number of shared blocks dirtied by the statement.
+  float m_shared_blks_dirtied = 202;
+  // Total number of shared blocks written by the statement.
+  float m_shared_blks_written = 203;
+  // Total number of local block cache hits by the statement.
+  float m_local_blks_hit = 204;
+  // Total number of local blocks read by the statement.
+  float m_local_blks_read = 205;
+  // Total number of local blocks dirtied by the statement.
+  float m_local_blks_dirtied = 206;
+  // Total number of local blocks written by the statement.
+  float m_local_blks_written = 207;
+  // Total number of temp blocks read by the statement.
+  float m_temp_blks_read = 208;
+  // Total number of temp blocks written by the statement.
+  float m_temp_blks_written = 209;
+  // Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
+  float m_blk_read_time = 210;
+  // Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
+  float m_blk_write_time = 211;
```

Comment:
Let's start using this branch for now, and discuss in Minsk:
1. if we want to use embedded messages there;
2. if we want to split pmm-agent<->pmm-managed and pmm-managed<->pmm-agent protocols.

Reply:
ok